.github/workflows/master.yml

```diff
@@ -36,10 +36,11 @@ jobs:
           - java: 8
             spark: '3.0'
             spark-hadoop: '2.7'
+            profiles: '-Dspark.archive.mirror=https://archive.apache.org/dist/spark/spark-3.0.3 -Dspark.archive.name=spark-3.0.3-bin-hadoop2.7.tgz -Dmaven.plugin.scalatest.exclude.tags=org.apache.kyuubi.tags.ExtendedSQLTest,org.apache.kyuubi.tags.DeltaTest,org.apache.kyuubi.tags.IcebergTest'
```

Comment:
I think it's unnecessary, we have a profile named `spark-3.0` which will activate specific properties.

Reply:
Thanks, checked the pom.xml